ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2

```diff
@@ -1,22 +1,26 @@
-{% if aaa_accounting is defined and aaa_accounting is not none %}
+{% if aaa_accounting is arista.avd.defined %}
 
 ## AAA Accounting
 
 ### AAA Accounting Summary
 
 | Type | Sub-type | Record | Accounting Stores | Logging |
```

Comment:
Try to align the headings to the variables.
Ex.
`Sub-Type` -> `Commands`
`Record` -> `Record type`
`Accounting Stores` -> `Group`